src/acc/generate_chunk.cpp

```diff
@@ -114,12 +115,25 @@ void generate_chunk(const int tile, global_variables &globals) {
     const double *state_radius = state_radius_buffer.data;
     const int *state_geometry = state_geometry_buffer.data;
 
-#pragma acc parallel loop gang worker vector collapse(2) clover_use_target(globals.context.use_target)                                 \
+    // We always want to time this, even though it's during startup, when most
+    // of the timers are turned off
+    double start_time = timer();
```

Comment:
For OpenMP/OpenACC this seems to be not including the time to copy from `global.config.states` into the host staging buffer. However, in other models we are including that time. 

I think its easier to move the `start_time` to the top to make it consistent with the other models. Otherwise it will be very hard to separate allocation and then the copy with Kokkos. But we should ask the entire group.

Reply:
Makes sense, I'll make that change